src/image_builder.py

```diff
@@ -110,6 +110,16 @@ def image_builder(buildspec):
                 labels[var] = file_name
                 labels[f"{var}_URI"] = uri
 
+        if str(BUILDSPEC["framework"]).startswith("huggingface"):
+            if image_config.get("transformers_version"):
```

Comment:
QQ: Is there a way to implement a "membership-check" for the key, like `if "transformers_version" in image_config`?

This is generally done for python classes through https://docs.python.org/3/reference/datamodel.html#object.__contains__